content/2021-12-01-brigade-2-goes-ga.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Brigade 2 Goes GA"
+slug: "2021-12-01-brigade-2-goes-ga"
+description: "Brigade 2 is now generally available"
+date: "2021-12-01"
+author: "Kent Rancourt"
+summary: Brigade 2 is now generally available.
+---
+
+Alright, Brigadiers! This is the day we've all been waiting for. Brigade 2 is generally available! If you're ready to get started, head on over to our [QuickStart](https://docs.brigade.sh/intro/quickstart/).
+
+But that's not all!
+
+As promised in our previous post, we've given some overdue love to Brigade 1.x as it enters a six month maintenance window. Yesterday, we released v1.9.0 of our Helm chart. It installs the existing Brigade v1.4.0 software, but addresses the deprecation or removal or certain beta APIs from newer releases of Kubernetes, making v1.4.0 installable, for the first time, on the newest, shiniest clusters. Hot on its heels, we've also released Brigade v1.5.0, which primarily remediates high severity vulnerabilities in the worker's JavaScript dependencies. It's installable using Helm chart v1.10.0.
```

Comment:
> our previous post

make that a link?

Reply:
> or removal or certain beta APIs

or removal **of** certain beta APIs

Reply:
> Hot on its heels

Idiomatic and may confuse ESL speakers - maybe "Also" or "Following that"?

Reply:
> It's installable

I might rephrase this as "Brigade v1.5.0 is installable" to keep people from needing to dereference that pointer.

Reply:
All of these should be fixed in 907e114c629a5a6a340e71d9c3c65bd17f869dd8